### PR TITLE
fix kmd loading and improve docs

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Proposed Changes

- fix KMD Credentials loading
- Updated algob confi docs
- Updated executeTx docs

### CODE REVIEW

File: docs/guide/algob-config.md [hunk 1]
```diff
@@ -33,55 +33,61 @@ loggingEnabled?: boolean;
...
-To load it from ENV:
+You can specify **Algod** [Network Credentials](https://algobuilder.dev/api/algob/interfaces/types.NetworkCredentials.html) manually or you can load it from ENV.
 
-- Method 1
+- Method 1: Specify manually:
 
-  - To add `ALGOD_ADDR` and `ALGOD_TOKEN` in env, you can use the following commands in terminal:
+        let algodCred = {host: "127.0.0.1", port: 4001, token: "c3e..."}
+
+- Method 2
```
Comment (rmeena840) [inline docs/guide/algob-config.md:46]: Can we also give this method also a name similar to method 1?

File: docs/guide/algob-config.md [hunk 2]
```diff
@@ -33,55 +33,61 @@ loggingEnabled?: boolean;
...
+Similarly, **KMD** uses [Network Credentials](https://algobuilder.dev/api/algob/interfaces/types.NetworkCredentials.html) object:
 
-- Method 1
+- Method 1: Specify manually:
 
-  - To add `KMD_ADDR` and `KMD_TOKEN` in env, you can use the following commands in terminal:
+        let kmdCred = {host: "127.0.0.1", port: 7833, token: "c3e..."}
 
-        export KMD_ADDR = "127.0.0.1:4001"
+- Method 2
```
Comment (rmeena840) [inline docs/guide/algob-config.md:73]: Same here

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed.

File: docs/guide/algob-config.md (hunk 1 above)
Comment (robert-zaremba) [reply docs/guide/algob-config.md]: OK

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**